Author: Andreas Tille <tille@debian.org>
        Wout B <wbertrums@gmail.com>
Last-Update: Thu, 19 Oct 2017 14:58:04 +0200
Description: There were some problems to find QDebug but this was
 most probably due to configuration issues.
 .
 This patch is deactivated for the moment

--- a/src/beads.h
+++ b/src/beads.h
@@ -16,7 +16,7 @@
 #ifndef BEADS_H_
 #define BEADS_H_
 
-#include <QDebug>
+#include <QtCore/qdebug.h>
 #include <QApplication>
 #include "config.h"
 
--- a/src/images/imageIntensity.h
+++ b/src/images/imageIntensity.h
@@ -1,7 +1,7 @@
 #ifndef IMAGEINTENSITY_H_
 #define IMAGEINTENSITY_H_
 
-#include <QDebug>
+#include <QtCore/qdebug.h>
 #include <QString>
 
 #include "imageCode.h"
--- a/src/qtbeads/qtbeads.cpp
+++ b/src/qtbeads/qtbeads.cpp
@@ -1,4 +1,4 @@
-#include <QDebug>
+#include <QtCore/qdebug.h>
 #include <QApplication>
 #include "../config.h"
 #include "main_window.h"
--- a/src/spotDocument.h
+++ b/src/spotDocument.h
@@ -1,7 +1,7 @@
 #ifndef SPOTDOCUMENT_H_
 #define SPOTDOCUMENT_H_
 
-#include <QDebug>
+#include <QtCore/qdebug.h>
 #include <QFile>
 #include <QTextStream>
 
--- a/src/qtbeads/properties.cpp
+++ b/src/qtbeads/properties.cpp
@@ -7,7 +7,7 @@
 
 #include "properties.h"
 #include "qtbeads_error.h"
-#include <QDebug>
+#include <QtCore/qdebug.h>
 #include <string>
 #include <iostream>
 #include <fstream>
